src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -76,12 +78,12 @@ public class TypicalPersons {
     // Manually added - Person's details found in {@code CommandTestUtil}
     public static final Person AMY = new PersonBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
             .withNric(VALID_NRIC_AMY).withDateOfBirth(VALID_DOB_AMY).withGender(VALID_GENDER_AMY)
-            .withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY).withTags(VALID_TAG_FRIEND)
-            .build();
+            .withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY)
+            .withPriority(VALID_PRIORITY_AMY).withMedCons(VALID_MEDCON).build();
     public static final Person BOB = new PersonBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
             .withNric(VALID_NRIC_BOB).withDateOfBirth(VALID_DOB_BOB).withGender(VALID_GENDER_BOB)
             .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB)
-            .build();
+            .withPriority(VALID_PRIORITY_BOB).build();
```

Comment:
I think it would be better to not include `Priority` here since both persons are presumed to have been 'just added' and no other command has been applied to them.

Not quite sure regarding this 🤔

Reply:
I will leave it in case we need to test a command that tests the existing priority.

I suspect it still works without `.withPriority()` because the `Priority`-related checks right now do not check the initial state, only the new state.

This will become an issue with `ListPriority` and `Find` when testing for different `Priority`s